### PR TITLE
#TechSolidarity

### PR DESCRIPTION
**Pull request by nataliyaned**

### CODE REVIEW

**Comment by nataliyaned** [comment]
Can't figure out why this one isn't going through :/ Says I'm missing a comma on line 20 but looks alright to me.

File: actions/01nn.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "online"
+    ],
+    "companies": [
+        "amazon"
+    ],
+    "workers": null,
+    "tags": [
+        "industry_solidarity",
+        "coworker_solidarity",
+    "coronavirus"
```
Comment (shushugah) [inline actions/01nn.json:31]: is this possibly supposed to be indented? I don't believe so

File: actions/01nn.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    ],
+    "struggles": [
+        "ethics",
+        "working_conditions",
+        "unfair_labor_practices"
+    ],
+    "employment_types": [
+        "white_collar_workers"
+    ],
+    "description": "As a gesture of solidarity, tech workers have started tweeting with the hashtags #TechSolidarity, #AmazonStrike, #HuelgaAmazon to indicate support for striking Amazon workers. The Twitter page, #TechSpeaksOut, was reportedly designed to exert public pressure tech corporations. In addition to taking to Twitter, activists have started GoFundMes and mutual aid campaigns to benefit workers.",
```
Comment (shushugah) [inline actions/01nn.json:20]: `The Twitter page, #TechSpeaksOut` should be `The Twitter handle @TechSpeaksOut`

File: actions/01nn.json (hunk 1 above)
Comment (nataliyaned) [reply actions/01nn.json]: Ahhh thank you! That worked

**Pull request merged**